iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerWithEngine.java

```diff
@@ -826,4 +866,31 @@ private StringBuilder buildComplexTypeInnerQuery(Object field, Type type) {
     }
     return query;
   }
+
+  private void validateBasicStats(Table table, String tableName) {
```

Comment:
Do we need the tableName param here? Can we use table.name()?

Reply:
I would rather keep the separate tableName param. table.name() returns it in `hive.default.customers` format.